lib/google_sheet.rb

```diff
@@ -14,8 +28,16 @@ def initialize(sheets_api: Google::Apis::SheetsV4::SheetsService.new, scope: :re
               Google::Apis::SheetsV4::AUTH_SPREADSHEETS
             end
 
+    json_key_io = if !GoogleSheet::GOOGLE_SERVICE_ACCOUNT_KEY.nil?
```

Comment:
will `if GoogleSheet::GOOGLE_SERVICE_ACCOUNT_KEY` not work here? (it's going to be either nil or truthy, right?)

Reply:
Agreed 👍 removed and tested locally, works ok.